tox.ini

```diff
@@ -30,6 +32,7 @@ deps =
 deps =
     {[base]deps}
     {py27,py37,pypy,pypy3}: twisted
+    {py35,py36,py37,py38,pypy3}: asgiref
```

Comment:
We only test the latest Python 3, which is 3.7 here.

Reply:
I'm not sure I understand.

It does seem like Travis runs through several Python 3 versions here: https://travis-ci.org/prometheus/client_python/builds/652096731 ?

Reply:
Not for things that add dependencies, we only test them on the latest.

Why does this pull in a dependency?

Reply:
The dependency is to get `asgiref.testing`; https://github.com/django/asgiref/blob/master/asgiref/testing.py

It's the reference support library for assisting in testing asyncronous applications within syncronous test-frameworks.

Reply:
You still need to trim down the line here.

Reply:
Done